packages/clay-card/src/CardHorizontal.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+import Context from './Context';
+
+interface IProps extends React.BaseHTMLAttributes<HTMLDivElement> {
+	/**
+	 * Flag that indicates if the card can be selectable.
+	 */
+	selectable?: boolean;
+}
+
+const ClayCardHorizontalBody: React.FunctionComponent<React.HTMLAttributes<
+	HTMLDivElement
+>> = ({children, className, ...otherProps}) => (
+	<div
+		className={classNames('card card-horizontal', className)}
+		{...otherProps}
+	>
+		{children}
+	</div>
+);
+
+export const ClayCardHorizontal: React.FunctionComponent<IProps> & {
+	Body: typeof ClayCardHorizontalBody;
+} = ({children, className, selectable, ...otherProps}) => (
+	<Context.Provider value={{horizontal: true, interactive: false}}>
```

Comment:
All components are using `Context` just to maintain compatibility and also because we use it directly on high-level components, we want to try to get rid of it in the future.